crates/uv-cli/src/lib.rs

```diff
@@ -285,6 +286,11 @@ pub enum Commands {
     /// e.g., if `pypy` is requested, uv will first check if the virtual
     /// environment contains a PyPy interpreter then check if each executable in
     /// the path is a PyPy interpreter.
+    ///
+    /// uv supports discovering CPython, PyPy, and GraalPy interpreters.
```

Comment:
I'm mixed on mentioning this (GraalPy is pretty rare).

Reply:
Mentioning our support or GraalPy in general? This is relatively important because if we don't explicitly support the implementation we will either error or ignore it.